### PR TITLE
Swiss curves page text update

### LINKED ISSUE #800: Swiss curves page text update
Up until recently, Swiss wastewater curves have been published both on the cov-spectrum website and on a dedicated ETHZ CBG webpage (https://bsse.ethz.ch/cbg/research/computational-virology/sarscov2-variants-wastewater-surveillance.html). After discussing with all parties involved it was decided to deprecate the CBG webpage and publish only on cov-spectrum.
Important text regarding methods, people involved and links should be included in cov-spectrum from the CBG page to finalise the transition.

Details on the final content that reached consensus can be found on a [dedicated Google document](https://docs.google.com/document/d/1149ReJz4j_h1UjLitJUHqYYC2mzOxX6uzJrj526MzYk)

To summarise the changes listed in the linked Google document:
- expand the introduction before the curves with details on the challenges of wastewater detection
- Include a `data sources` section
- Include the links to the video presentation of the project
- Include an `acknowledgments` section
- Include a `funding` section
- Include a `contacts` section

The hyperlinks to include are available either embedded in the text or as comments in the document.

### PR DESCRIPTION
**Pull request by JonasKellerer**
Resolves #800

### CODE REVIEW

**Comment by JonasKellerer** [comment]
I added the relevant text, but we could discus about the design :)

File: src/models/wasteWater/story/WasteWaterStoryPage.tsx [hunk 1]
```diff
@@ -15,29 +15,136 @@ export const WasteWaterStoryPage = () => {
...
+      </p>
+    </div>
+  );
+};
+
+const Acknowledgements = () => {
+  return (
+    <div>
+      <h2>Acknowledgements</h2>
+      <h3>Computational Biology Group (CBG) ETH Zürich / Swiss Institute of Bioinformatics</h3>
```
Comment (chaoran-chen) [inline src/models/wasteWater/story/WasteWaterStoryPage.tsx]: "Computational Biology Group (CBG) ETH Zürich" should point to https://bsse.ethz.ch/cbg

"Swiss Institute of Bioinformatics" should point to https://www.sib.swiss/niko-beerenwinkel-group

File: src/models/wasteWater/story/WasteWaterStoryPage.tsx [hunk 2]
```diff
@@ -15,29 +15,136 @@ export const WasteWaterStoryPage = () => {
...
+      <Authors
+        authors={
+          'Katharina Jahn, Pelin Burcak Icer, David Dreifuss, Ivan Topolsky, Lara Fuhrmann, Kim Philipp Jablonski, Anika John, Niko Beerenwinkel'
+        }
+      />
+      <h3>Computational Evolution (cEvo) ETH Zürich / Swiss Institute of Bioinformatics</h3>
```
Comment (chaoran-chen) [inline src/models/wasteWater/story/WasteWaterStoryPage.tsx]: "Computational Evolution (cEvo) ETH Zürich" should point to https://bsse.ethz.ch/cevo

"Swiss Institute of Bioinformatics" should point to https://www.sib.swiss/tanja-stadler-group

File: src/models/wasteWater/story/WasteWaterStoryPage.tsx [hunk 3]
```diff
@@ -15,29 +15,136 @@ export const WasteWaterStoryPage = () => {
...
+      <Authors authors={'Chaoran Chen, Sarah Nadeau, Tanja Stadler'} />
+      <h3>
+        <ExternalLink url={'https://www.nexus.ethz.ch/'}>NEXUS Personalized Health Technologies</ExternalLink>
+        , ETH Zürich / Swiss Institute of Bioinformatics
```
Comment (chaoran-chen) [inline src/models/wasteWater/story/WasteWaterStoryPage.tsx]: "Swiss Institute of Bioinformatics" should point to https://www.sib.swiss/daniel-stekhoven-group

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
There are a few more links that should be added. They are in the document as comments which you couldn't see as a guest:



I added the links in the comments in code

**Comment by chaoran-chen** [comment]
@mcarrara-bioinfo, what do you think? Please let us know if this is as you expected!

**Comment by mcarrara-bioinfo** [comment]
> @mcarrara-bioinfo, what do you think? Please let us know if this is as you expected!

@chaoran-chen The preview does not work for me. The page itself loads, but it hangs on loading the actual content, leaving only header, footer and loading animation. I left it for 10 minutes and saw no change. Do I need a Vercel account to access?

**Comment by chaoran-chen** [comment]
Hi @mcarrara-bioinfo, sorry, CoV-Spectrum was down for 30 minutes and is just back. Can you please try again?

**Comment by mcarrara-bioinfo** [comment]
> Hi @mcarrara-bioinfo, sorry, CoV-Spectrum was down for 30 minutes and is just back. Can you please try again?
@chaoran-chen thank you, after your message I tried again and it worked.

Dear all, both CBG and NEXUS reviewed the preview and everything is approved as-is. As far as we are concerned, you have greenlight to deploy the changes as soon as you see fit.

Thank you for your efforts.

**Pull request merged**